[2023-12-18T00:00:06.983+0000] {processor.py:157} INFO - Started process (PID=1241) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:00:06.984+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:00:06.984+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:00:06.984+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:00:06.988+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:00:06.987+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:00:06.988+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:00:06.988+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:00:06.988+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:00:06.998+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2023-12-18T00:00:59.105+0000] {processor.py:157} INFO - Started process (PID=1351) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:00:59.107+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:00:59.108+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:00:59.107+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:00:59.111+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:00:59.110+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:00:59.111+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:00:59.111+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:00:59.111+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:00:59.123+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-18T00:01:50.032+0000] {processor.py:157} INFO - Started process (PID=1403) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:01:50.033+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:01:50.034+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:01:50.034+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:01:50.038+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:01:50.037+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:01:50.038+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:01:50.038+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:01:50.039+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:01:50.051+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-12-18T00:02:41.004+0000] {processor.py:157} INFO - Started process (PID=1455) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:02:41.024+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:02:41.026+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:02:41.025+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:02:41.035+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:02:41.034+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:02:41.035+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:02:41.035+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:02:41.035+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:02:41.047+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2023-12-18T00:03:27.129+0000] {processor.py:157} INFO - Started process (PID=1503) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:03:27.130+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:03:27.131+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:03:27.131+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:03:27.134+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:03:27.134+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:03:27.135+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:03:27.135+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:03:27.135+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:03:27.144+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2023-12-18T00:04:17.846+0000] {processor.py:157} INFO - Started process (PID=1553) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:04:17.848+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:04:17.848+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:04:17.848+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:04:17.852+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:04:17.851+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:04:17.852+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:04:17.852+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:04:17.852+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:04:17.864+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-18T00:05:12.792+0000] {processor.py:157} INFO - Started process (PID=1609) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:05:12.794+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:05:12.794+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:05:12.794+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:05:12.798+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:05:12.797+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:05:12.798+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:05:12.798+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:05:12.798+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:05:12.812+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-18T00:06:03.269+0000] {processor.py:157} INFO - Started process (PID=1660) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:06:03.270+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:06:03.271+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:06:03.271+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:06:03.275+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:06:03.274+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:06:03.276+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:06:03.276+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:06:03.276+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:06:03.288+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-12-18T00:06:58.212+0000] {processor.py:157} INFO - Started process (PID=1716) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:06:58.213+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:06:58.214+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:06:58.214+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:06:58.218+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:06:58.217+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:06:58.218+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:06:58.218+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:06:58.218+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:06:58.230+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-18T00:12:46.578+0000] {processor.py:157} INFO - Started process (PID=880) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:12:46.579+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:12:46.580+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:12:46.580+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:12:46.584+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:12:46.584+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:12:46.584+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:12:46.584+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:12:46.585+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:12:46.597+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-18T00:13:38.097+0000] {processor.py:157} INFO - Started process (PID=931) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:13:38.099+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:13:38.100+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:13:38.099+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:13:38.104+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:13:38.103+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:13:38.104+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:13:38.104+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:13:38.105+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:13:38.127+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-12-18T00:14:23.240+0000] {processor.py:157} INFO - Started process (PID=978) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:14:23.242+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:14:23.243+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:14:23.242+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:14:23.246+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:14:23.246+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:14:23.247+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:14:23.247+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:14:23.247+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:14:23.256+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-18T00:15:11.964+0000] {processor.py:157} INFO - Started process (PID=1027) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:15:11.965+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:15:11.966+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:15:11.966+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:15:11.970+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:15:11.969+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:15:11.971+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:15:11.970+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:15:11.971+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:15:11.991+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-12-18T00:16:03.616+0000] {processor.py:157} INFO - Started process (PID=1084) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:16:03.618+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:16:03.618+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:16:03.618+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:16:03.622+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:16:03.621+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:16:03.622+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:16:03.622+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:16:03.623+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:16:03.634+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-18T00:16:55.188+0000] {processor.py:157} INFO - Started process (PID=1139) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:16:55.189+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:16:55.190+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:16:55.189+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:16:55.193+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:16:55.192+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:16:55.193+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:16:55.193+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:16:55.193+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:16:55.203+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2023-12-18T00:17:49.739+0000] {processor.py:157} INFO - Started process (PID=1196) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:17:49.740+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:17:49.741+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:17:49.740+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:17:49.745+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:17:49.744+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:17:49.745+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:17:49.745+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:17:49.745+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:17:49.759+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-12-18T00:18:43.009+0000] {processor.py:157} INFO - Started process (PID=1249) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:18:43.010+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:18:43.011+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:18:43.011+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:18:43.016+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:18:43.015+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:18:43.017+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:18:43.017+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:18:43.017+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:18:43.033+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-12-18T00:19:30.247+0000] {processor.py:157} INFO - Started process (PID=1298) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:19:30.249+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:19:30.249+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:19:30.249+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:19:30.253+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:19:30.252+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:19:30.253+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:19:30.253+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:19:30.253+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:19:30.264+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2023-12-18T00:20:23.120+0000] {processor.py:157} INFO - Started process (PID=1350) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:20:23.121+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:20:23.122+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:20:23.122+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:20:23.126+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:20:23.125+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:20:23.126+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:20:23.126+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:20:23.127+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:20:23.143+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-12-18T00:21:20.111+0000] {processor.py:157} INFO - Started process (PID=1407) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:21:20.114+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:21:20.115+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:21:20.115+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:21:20.121+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:21:20.120+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:21:20.121+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:21:20.121+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:21:20.121+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:21:20.144+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-12-18T00:22:13.559+0000] {processor.py:157} INFO - Started process (PID=1461) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:22:13.560+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:22:13.561+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:22:13.561+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:22:13.565+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:22:13.564+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:22:13.565+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:22:13.565+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:22:13.565+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:22:13.577+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-18T00:23:02.695+0000] {processor.py:157} INFO - Started process (PID=1511) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:23:02.696+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:23:02.697+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:23:02.697+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:23:02.701+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:23:02.700+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:23:02.701+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:23:02.701+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:23:02.701+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:23:02.713+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-18T00:23:55.070+0000] {processor.py:157} INFO - Started process (PID=1564) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:23:55.071+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:23:55.072+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:23:55.071+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:23:55.076+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:23:55.075+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:23:55.076+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:23:55.076+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:23:55.076+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:23:55.089+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-18T00:24:44.945+0000] {processor.py:157} INFO - Started process (PID=1615) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:24:44.946+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:24:44.947+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:24:44.947+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:24:44.951+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:24:44.950+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:24:44.951+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:24:44.951+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:24:44.951+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:24:44.966+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-12-18T00:25:39.960+0000] {processor.py:157} INFO - Started process (PID=1710) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:25:39.961+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:25:39.962+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:25:39.962+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:25:39.966+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:25:39.965+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:25:39.966+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:25:39.966+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:25:39.966+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:25:39.980+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-18T00:26:26.661+0000] {processor.py:157} INFO - Started process (PID=1760) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:26:26.662+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:26:26.663+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:26:26.663+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:26:26.667+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:26:26.666+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:26:26.668+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:26:26.668+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:26:26.668+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:26:26.679+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-18T00:27:18.095+0000] {processor.py:157} INFO - Started process (PID=1812) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:27:18.096+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:27:18.097+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:27:18.096+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:27:18.100+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:27:18.099+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:27:18.100+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:27:18.100+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:27:18.101+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:27:18.130+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2023-12-18T00:28:12.495+0000] {processor.py:157} INFO - Started process (PID=1865) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:28:12.498+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:28:12.499+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:28:12.498+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:28:12.502+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:28:12.502+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:28:12.503+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:28:12.502+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:28:12.503+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:28:12.514+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-12-18T00:29:04.484+0000] {processor.py:157} INFO - Started process (PID=1918) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:29:04.486+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:29:04.487+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:29:04.486+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:29:04.492+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:29:04.491+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:29:04.492+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:29:04.492+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:29:04.493+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:29:04.507+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-12-18T00:29:59.062+0000] {processor.py:157} INFO - Started process (PID=1975) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:29:59.065+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:29:59.066+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:29:59.066+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:29:59.071+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:29:59.070+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:29:59.071+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:29:59.071+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:29:59.071+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:29:59.085+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-12-18T00:30:48.903+0000] {processor.py:157} INFO - Started process (PID=2025) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:30:48.910+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:30:48.911+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:30:48.911+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:30:48.917+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:30:48.916+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:30:48.917+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:30:48.917+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:30:48.917+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:30:48.930+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-12-18T00:31:43.022+0000] {processor.py:157} INFO - Started process (PID=2080) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:31:43.023+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:31:43.024+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:31:43.024+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:31:43.028+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:31:43.027+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:31:43.028+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:31:43.028+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:31:43.029+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:31:43.040+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-18T00:32:35.049+0000] {processor.py:157} INFO - Started process (PID=2133) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:32:35.050+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:32:35.051+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:32:35.051+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:32:35.055+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:32:35.054+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:32:35.055+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:32:35.055+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:32:35.055+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:32:35.065+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2023-12-18T00:33:25.781+0000] {processor.py:157} INFO - Started process (PID=2261) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:33:25.813+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:33:25.814+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:33:25.814+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:33:25.819+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:33:25.818+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:33:25.819+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:33:25.819+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:33:25.819+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:33:25.830+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.052 seconds
[2023-12-18T00:34:18.019+0000] {processor.py:157} INFO - Started process (PID=2319) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:34:18.020+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:34:18.022+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:34:18.021+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:34:18.026+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:34:18.025+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:34:18.026+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:34:18.026+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:34:18.027+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:34:18.041+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-12-18T00:35:09.232+0000] {processor.py:157} INFO - Started process (PID=2373) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:35:09.235+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:35:09.236+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:35:09.235+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:35:09.244+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:35:09.240+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:35:09.244+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:35:09.244+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:35:09.244+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:35:09.261+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-12-18T00:35:58.081+0000] {processor.py:157} INFO - Started process (PID=2426) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:35:58.083+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:35:58.084+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:35:58.084+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:35:58.087+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:35:58.087+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:35:58.088+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:35:58.088+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:35:58.088+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:35:58.110+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-12-18T00:36:47.409+0000] {processor.py:157} INFO - Started process (PID=2480) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:36:47.411+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:36:47.412+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:36:47.411+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:36:47.420+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:36:47.419+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:36:47.420+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:36:47.420+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:36:47.420+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:36:47.446+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2023-12-18T00:37:41.710+0000] {processor.py:157} INFO - Started process (PID=2540) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:37:41.712+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:37:41.713+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:37:41.712+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:37:41.716+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:37:41.716+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:37:41.716+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:37:41.716+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:37:41.716+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:37:41.726+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2023-12-18T00:38:30.768+0000] {processor.py:157} INFO - Started process (PID=2593) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:38:30.769+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:38:30.770+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:38:30.770+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:38:30.773+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:38:30.773+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:38:30.773+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:38:30.773+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:38:30.774+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:38:30.783+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2023-12-18T00:39:26.236+0000] {processor.py:157} INFO - Started process (PID=2653) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:39:26.237+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:39:26.238+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:39:26.238+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:39:26.242+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:39:26.241+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:39:26.242+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:39:26.242+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:39:26.242+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:39:26.253+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-18T00:40:19.408+0000] {processor.py:157} INFO - Started process (PID=2717) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:40:19.409+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:40:19.410+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:40:19.410+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:40:19.413+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:40:19.413+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:40:19.414+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:40:19.413+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:40:19.414+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:40:19.424+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2023-12-18T00:41:12.222+0000] {processor.py:157} INFO - Started process (PID=2771) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:41:12.223+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:41:12.224+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:41:12.223+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:41:12.231+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:41:12.228+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:41:12.231+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:41:12.231+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:41:12.232+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:41:12.242+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-12-18T00:42:33.840+0000] {processor.py:157} INFO - Started process (PID=2848) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:42:33.843+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:42:33.844+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:42:33.843+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:42:33.848+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:42:33.847+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:42:33.848+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:42:33.848+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:42:33.848+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:42:33.859+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-18T00:43:22.785+0000] {processor.py:157} INFO - Started process (PID=2896) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:43:22.786+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:43:22.787+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:43:22.787+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:43:22.797+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:43:22.794+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:43:22.797+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:43:22.797+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:43:22.798+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:43:22.821+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2023-12-18T00:44:17.281+0000] {processor.py:157} INFO - Started process (PID=2950) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:44:17.283+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:44:17.284+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:44:17.283+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:44:17.287+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:44:17.286+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:44:17.287+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:44:17.287+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:44:17.287+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:44:17.297+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2023-12-18T00:45:14.449+0000] {processor.py:157} INFO - Started process (PID=3023) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:45:14.455+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:45:14.456+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:45:14.455+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:45:14.460+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:45:14.459+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:45:14.460+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:45:14.460+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:45:14.461+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:45:14.472+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-12-18T00:46:08.723+0000] {processor.py:157} INFO - Started process (PID=3083) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:46:08.724+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:46:08.725+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:46:08.724+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:46:08.729+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:46:08.728+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:46:08.729+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:46:08.729+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:46:08.730+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:46:08.741+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-18T00:47:04.908+0000] {processor.py:157} INFO - Started process (PID=3139) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:47:04.910+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:47:04.911+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:47:04.910+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:47:04.914+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:47:04.913+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:47:04.914+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:47:04.914+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:47:04.915+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:47:04.926+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-18T00:47:59.927+0000] {processor.py:157} INFO - Started process (PID=3210) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:47:59.928+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:47:59.928+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:47:59.928+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:47:59.932+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:47:59.931+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:47:59.932+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:47:59.932+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:47:59.932+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:47:59.943+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2023-12-18T00:48:58.020+0000] {processor.py:157} INFO - Started process (PID=3269) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:48:58.022+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:48:58.023+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:48:58.022+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:48:58.029+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:48:58.027+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:48:58.029+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:48:58.029+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:48:58.029+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:48:58.040+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-18T00:49:59.077+0000] {processor.py:157} INFO - Started process (PID=3330) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:49:59.078+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:49:59.079+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:49:59.079+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:49:59.084+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:49:59.083+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:49:59.085+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:49:59.084+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:49:59.085+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:49:59.095+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-12-18T00:50:53.189+0000] {processor.py:157} INFO - Started process (PID=3386) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:50:53.191+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:50:53.192+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:50:53.191+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:50:53.198+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:50:53.198+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:50:53.199+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:50:53.199+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:50:53.199+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:50:53.224+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2023-12-18T00:51:51.613+0000] {processor.py:157} INFO - Started process (PID=3446) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:51:51.617+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:51:51.619+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:51:51.618+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:51:51.623+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:51:51.622+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:51:51.623+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:51:51.623+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:51:51.624+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:51:51.634+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-12-18T00:52:47.219+0000] {processor.py:157} INFO - Started process (PID=3504) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:52:47.221+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:52:47.222+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:52:47.222+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:52:47.226+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:52:47.225+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:52:47.226+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:52:47.226+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:52:47.227+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:52:47.237+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-18T00:53:44.544+0000] {processor.py:157} INFO - Started process (PID=3562) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:53:44.545+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:53:44.546+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:53:44.546+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:53:44.554+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:53:44.550+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:53:44.555+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:53:44.555+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:53:44.555+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:53:44.566+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-12-18T00:54:41.452+0000] {processor.py:157} INFO - Started process (PID=3620) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:54:41.454+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:54:41.456+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:54:41.455+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:54:41.459+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:54:41.458+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:54:41.459+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:54:41.459+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:54:41.460+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:54:41.480+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-12-18T00:55:40.280+0000] {processor.py:157} INFO - Started process (PID=3684) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:55:40.282+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:55:40.283+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:55:40.282+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:55:40.286+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:55:40.285+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:55:40.286+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:55:40.286+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:55:40.286+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:55:40.302+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-12-18T00:56:33.331+0000] {processor.py:157} INFO - Started process (PID=3740) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:56:33.333+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:56:33.334+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:56:33.333+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:56:33.337+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:56:33.337+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:56:33.338+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:56:33.338+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:56:33.338+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:56:33.349+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-18T00:57:30.705+0000] {processor.py:157} INFO - Started process (PID=3797) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:57:30.706+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:57:30.707+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:57:30.707+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:57:30.711+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:57:30.710+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:57:30.711+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:57:30.711+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:57:30.711+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:57:30.732+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-12-18T00:58:27.967+0000] {processor.py:157} INFO - Started process (PID=3856) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:58:27.969+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:58:27.970+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:58:27.969+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:58:27.974+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:58:27.973+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:58:27.974+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:58:27.974+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:58:27.974+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:58:27.985+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-18T00:59:21.331+0000] {processor.py:157} INFO - Started process (PID=3911) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:59:21.333+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T00:59:21.334+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:59:21.334+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:59:21.339+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:59:21.338+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T00:59:21.339+0000] {logging_mixin.py:154} INFO - [2023-12-18T00:59:21.339+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T00:59:21.340+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T00:59:21.353+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-12-18T01:00:19.913+0000] {processor.py:157} INFO - Started process (PID=3970) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:00:19.915+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:00:19.916+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:00:19.916+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:00:19.920+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:00:19.920+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:00:19.921+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:00:19.921+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:00:19.921+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:00:19.932+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-18T01:01:20.470+0000] {processor.py:157} INFO - Started process (PID=4030) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:01:20.489+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:01:20.490+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:01:20.490+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:01:20.494+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:01:20.493+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:01:20.494+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:01:20.494+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:01:20.494+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:01:20.504+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2023-12-18T01:02:16.789+0000] {processor.py:157} INFO - Started process (PID=4088) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:02:16.791+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:02:16.792+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:02:16.792+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:02:16.797+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:02:16.797+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:02:16.798+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:02:16.798+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:02:16.798+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:02:16.809+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-18T01:03:12.856+0000] {processor.py:157} INFO - Started process (PID=4144) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:03:12.857+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:03:12.859+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:03:12.858+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:03:12.863+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:03:12.862+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:03:12.864+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:03:12.863+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:03:12.864+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:03:12.878+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-12-18T01:06:00.854+0000] {processor.py:157} INFO - Started process (PID=4334) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:06:00.855+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:06:00.856+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:06:00.856+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:06:00.860+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:06:00.859+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:06:00.860+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:06:00.860+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:06:00.860+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:06:00.871+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-18T01:09:13.651+0000] {processor.py:157} INFO - Started process (PID=4533) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:09:13.653+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:09:13.654+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:09:13.654+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:09:13.659+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:09:13.658+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:09:13.659+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:09:13.659+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:09:13.659+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:09:13.671+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-12-18T01:10:12.758+0000] {processor.py:157} INFO - Started process (PID=4593) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:10:12.761+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:10:12.762+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:10:12.762+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:10:12.773+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:10:12.772+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:10:12.781+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:10:12.781+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:10:12.782+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:10:12.794+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2023-12-18T01:11:14.167+0000] {processor.py:157} INFO - Started process (PID=4656) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:11:14.169+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:11:14.170+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:11:14.169+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:11:14.174+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:11:14.173+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:11:14.174+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:11:14.174+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:11:14.174+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:11:14.186+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-18T01:14:07.441+0000] {processor.py:157} INFO - Started process (PID=4838) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:14:07.442+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:14:07.443+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:14:07.442+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:14:07.446+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:14:07.446+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:14:07.446+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:14:07.446+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:14:07.447+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:14:07.468+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-12-18T01:15:13.327+0000] {processor.py:157} INFO - Started process (PID=4906) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:15:13.333+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:15:13.334+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:15:13.333+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:15:13.338+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:15:13.337+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:15:13.338+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:15:13.338+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:15:13.338+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:15:13.351+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2023-12-18T01:16:01.433+0000] {processor.py:157} INFO - Started process (PID=4953) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:16:01.439+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:16:01.441+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:16:01.440+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:16:01.449+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:16:01.448+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:16:01.449+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:16:01.449+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:16:01.449+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:16:01.477+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
[2023-12-18T01:16:58.410+0000] {processor.py:157} INFO - Started process (PID=5011) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:16:58.419+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:16:58.421+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:16:58.420+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:16:58.425+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:16:58.424+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:16:58.425+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:16:58.425+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:16:58.425+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:16:58.439+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-12-18T01:17:57.619+0000] {processor.py:157} INFO - Started process (PID=5074) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:17:57.621+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:17:57.621+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:17:57.621+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:17:57.625+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:17:57.624+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:17:57.625+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:17:57.625+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:17:57.625+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:17:57.639+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-12-18T01:18:54.483+0000] {processor.py:157} INFO - Started process (PID=5135) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:18:54.486+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:18:54.487+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:18:54.486+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:18:54.506+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:18:54.504+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:18:54.507+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:18:54.507+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:18:54.508+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:18:54.538+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2023-12-18T01:19:55.956+0000] {processor.py:157} INFO - Started process (PID=5196) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:19:55.958+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:19:55.959+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:19:55.958+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:19:55.963+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:19:55.963+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:19:55.964+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:19:55.963+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:19:55.964+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:19:55.975+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-18T01:20:54.134+0000] {processor.py:157} INFO - Started process (PID=5254) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:20:54.136+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:20:54.137+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:20:54.137+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:20:54.143+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:20:54.142+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:20:54.143+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:20:54.143+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:20:54.143+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:20:54.155+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-12-18T01:21:50.748+0000] {processor.py:157} INFO - Started process (PID=5312) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:21:50.753+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:21:50.753+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:21:50.753+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:21:50.758+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:21:50.757+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:21:50.759+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:21:50.759+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:21:50.759+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:21:50.774+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-12-18T01:22:48.266+0000] {processor.py:157} INFO - Started process (PID=5372) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:22:48.268+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:22:48.268+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:22:48.268+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:22:48.272+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:22:48.271+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:22:48.272+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:22:48.272+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:22:48.272+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:22:48.284+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-18T01:23:48.939+0000] {processor.py:157} INFO - Started process (PID=5432) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:23:48.940+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:23:48.941+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:23:48.940+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:23:48.944+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:23:48.943+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:23:48.944+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:23:48.944+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:23:48.944+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:23:48.954+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2023-12-18T01:24:44.657+0000] {processor.py:157} INFO - Started process (PID=5489) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:24:44.658+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:24:44.659+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:24:44.659+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:24:44.663+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:24:44.662+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-18T01:24:44.663+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:24:44.663+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-18T01:24:44.663+0000] {processor.py:841} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:24:44.677+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-12-18T01:42:16.338+0000] {processor.py:157} INFO - Started process (PID=1392) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:42:16.340+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T01:42:16.341+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:42:16.340+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:42:16.350+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T01:42:16.385+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:42:16.385+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T01:42:16.398+0000] {logging_mixin.py:154} INFO - [2023-12-18T01:42:16.398+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T01:42:16.436+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.102 seconds
[2023-12-18T03:22:23.426+0000] {processor.py:157} INFO - Started process (PID=899) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:22:23.428+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:22:23.429+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:22:23.428+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:22:23.437+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:22:23.485+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:22:23.485+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:22:23.497+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:22:23.497+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:22:23.531+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.109 seconds
[2023-12-18T03:24:22.309+0000] {processor.py:157} INFO - Started process (PID=1093) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:24:22.315+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:24:22.316+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:24:22.316+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:24:22.338+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:24:22.380+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:24:22.379+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:24:22.390+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:24:22.390+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:24:22.411+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.105 seconds
[2023-12-18T03:25:17.627+0000] {processor.py:157} INFO - Started process (PID=1150) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:25:17.637+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:25:17.638+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:25:17.637+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:25:17.657+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:25:17.715+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:25:17.715+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:25:17.730+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:25:17.730+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:25:17.758+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.135 seconds
[2023-12-18T03:26:23.831+0000] {processor.py:157} INFO - Started process (PID=1207) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:26:23.834+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:26:23.835+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:26:23.834+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:26:23.849+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:26:24.040+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:26:24.040+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:26:24.050+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:26:24.049+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:26:24.077+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.249 seconds
[2023-12-18T03:27:25.173+0000] {processor.py:157} INFO - Started process (PID=1265) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:27:25.174+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:27:25.175+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:27:25.175+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:27:25.184+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:27:25.218+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:27:25.218+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:27:25.229+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:27:25.229+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:27:25.257+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.087 seconds
[2023-12-18T03:28:21.567+0000] {processor.py:157} INFO - Started process (PID=1333) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:28:21.569+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:28:21.571+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:28:21.570+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:28:21.585+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:28:21.631+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:28:21.631+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:28:21.644+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:28:21.643+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:28:21.667+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.104 seconds
[2023-12-18T03:29:19.892+0000] {processor.py:157} INFO - Started process (PID=1394) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:29:19.905+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:29:19.907+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:29:19.906+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:29:19.915+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:29:19.949+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:29:19.949+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:29:19.960+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:29:19.960+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:29:19.986+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.101 seconds
[2023-12-18T03:30:12.979+0000] {processor.py:157} INFO - Started process (PID=1454) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:30:12.980+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:30:12.981+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:30:12.980+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:30:12.990+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:30:13.027+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:30:13.027+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:30:13.039+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:30:13.039+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:30:13.063+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.089 seconds
[2023-12-18T03:30:58.417+0000] {processor.py:157} INFO - Started process (PID=1498) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:30:58.418+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:30:58.420+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:30:58.419+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:30:58.434+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:30:58.467+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:30:58.467+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:30:58.478+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:30:58.478+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:30:58.505+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.092 seconds
[2023-12-18T03:31:57.270+0000] {processor.py:157} INFO - Started process (PID=1555) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:31:57.273+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:31:57.275+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:31:57.274+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:31:57.286+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:31:57.327+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:31:57.323+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:31:57.348+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:31:57.348+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:31:57.371+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.105 seconds
[2023-12-18T03:34:51.815+0000] {processor.py:157} INFO - Started process (PID=967) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:34:51.820+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:34:51.821+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:34:51.821+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:34:51.829+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:34:51.865+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:34:51.865+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:34:51.876+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:34:51.875+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:34:51.903+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.091 seconds
[2023-12-18T03:35:41.470+0000] {processor.py:157} INFO - Started process (PID=1015) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:35:41.471+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:35:41.472+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:35:41.472+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:35:41.479+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:35:41.509+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:35:41.509+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:35:41.520+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:35:41.520+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:35:41.550+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.083 seconds
[2023-12-18T03:36:43.573+0000] {processor.py:157} INFO - Started process (PID=1075) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:36:43.574+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:36:43.575+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:36:43.575+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:36:43.588+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:36:43.622+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:36:43.622+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:36:43.836+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:36:43.835+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:36:43.876+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.306 seconds
[2023-12-18T03:37:37.824+0000] {processor.py:157} INFO - Started process (PID=1125) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:37:37.825+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:37:37.827+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:37:37.826+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:37:37.835+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:37:37.865+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:37:37.864+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:37:37.897+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:37:37.897+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:37:37.923+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.106 seconds
[2023-12-18T03:38:34.320+0000] {processor.py:157} INFO - Started process (PID=1185) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:38:34.322+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:38:34.323+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:38:34.323+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:38:34.333+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:38:34.372+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:38:34.372+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:38:34.383+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:38:34.383+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:38:34.427+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.112 seconds
[2023-12-18T03:39:33.816+0000] {processor.py:157} INFO - Started process (PID=1247) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:39:33.820+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:39:33.821+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:39:33.820+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:39:33.829+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:39:33.858+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:39:33.858+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:39:33.869+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:39:33.869+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:39:33.894+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.081 seconds
[2023-12-18T03:40:28.708+0000] {processor.py:157} INFO - Started process (PID=1306) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:40:28.709+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:40:28.710+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:40:28.710+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:40:28.718+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:40:28.761+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:40:28.761+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:40:28.776+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:40:28.776+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:40:28.811+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.106 seconds
[2023-12-18T03:41:24.391+0000] {processor.py:157} INFO - Started process (PID=1361) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:41:24.394+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:41:24.395+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:41:24.394+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:41:24.402+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:41:24.432+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:41:24.432+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:41:24.444+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:41:24.444+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:41:24.465+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.077 seconds
[2023-12-18T03:42:24.596+0000] {processor.py:157} INFO - Started process (PID=1422) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:42:24.599+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:42:24.600+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:42:24.599+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:42:24.609+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:42:24.646+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:42:24.646+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:42:24.657+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:42:24.657+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:42:24.680+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.087 seconds
[2023-12-18T03:43:20.920+0000] {processor.py:157} INFO - Started process (PID=1481) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:43:20.922+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:43:20.922+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:43:20.922+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:43:20.930+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:43:20.959+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:43:20.959+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:43:20.969+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:43:20.969+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:43:20.990+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
[2023-12-18T03:44:14.473+0000] {processor.py:157} INFO - Started process (PID=1538) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:44:14.475+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:44:14.476+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:44:14.475+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:44:14.485+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:44:14.534+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:44:14.534+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:44:14.557+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:44:14.557+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:44:14.587+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.126 seconds
[2023-12-18T03:45:14.325+0000] {processor.py:157} INFO - Started process (PID=1599) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:45:14.330+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:45:14.331+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:45:14.330+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:45:14.338+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:45:14.373+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:45:14.373+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:45:14.384+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:45:14.384+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:45:14.408+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.086 seconds
[2023-12-18T03:46:08.712+0000] {processor.py:157} INFO - Started process (PID=1659) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:46:08.713+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:46:08.714+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:46:08.714+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:46:08.722+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:46:08.760+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:46:08.760+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:46:08.773+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:46:08.773+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:46:08.799+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.091 seconds
[2023-12-18T03:48:46.840+0000] {processor.py:157} INFO - Started process (PID=885) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:48:46.876+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:48:46.876+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:48:46.876+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:48:46.889+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:48:46.924+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:48:46.923+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:48:46.934+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:48:46.934+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:48:46.955+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.119 seconds
[2023-12-18T03:49:37.160+0000] {processor.py:157} INFO - Started process (PID=1015) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:49:37.161+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:49:37.162+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:49:37.162+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:49:37.171+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:49:37.214+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:49:37.214+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:49:37.232+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:49:37.232+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:49:37.275+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.118 seconds
[2023-12-18T03:50:35.882+0000] {processor.py:157} INFO - Started process (PID=1071) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:50:35.883+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:50:35.884+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:50:35.884+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:50:35.891+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:50:35.922+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:50:35.922+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:50:36.096+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:50:36.096+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:50:36.117+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.238 seconds
[2023-12-18T03:53:05.755+0000] {processor.py:157} INFO - Started process (PID=880) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:53:05.757+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:53:05.758+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:53:05.758+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:53:05.767+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:53:05.811+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:53:05.811+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:53:05.824+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:53:05.823+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:53:05.856+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.104 seconds
[2023-12-18T03:55:57.988+0000] {processor.py:157} INFO - Started process (PID=1028) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:55:58.004+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:55:58.005+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:55:58.005+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:55:58.014+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:55:58.051+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:55:58.051+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:55:58.064+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:55:58.064+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:55:58.115+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.130 seconds
[2023-12-18T03:56:58.138+0000] {processor.py:157} INFO - Started process (PID=1104) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:56:58.151+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T03:56:58.152+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:56:58.151+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:56:58.161+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T03:56:58.195+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:56:58.195+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T03:56:58.206+0000] {logging_mixin.py:154} INFO - [2023-12-18T03:56:58.206+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T03:56:58.250+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.114 seconds
[2023-12-18T04:05:52.852+0000] {processor.py:157} INFO - Started process (PID=962) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T04:05:52.856+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T04:05:52.857+0000] {logging_mixin.py:154} INFO - [2023-12-18T04:05:52.857+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T04:05:52.866+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T04:05:52.903+0000] {logging_mixin.py:154} INFO - [2023-12-18T04:05:52.903+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T04:05:52.945+0000] {logging_mixin.py:154} INFO - [2023-12-18T04:05:52.945+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T04:05:52.987+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.138 seconds
[2023-12-18T04:08:16.853+0000] {processor.py:157} INFO - Started process (PID=1108) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T04:08:16.854+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T04:08:16.855+0000] {logging_mixin.py:154} INFO - [2023-12-18T04:08:16.855+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T04:08:16.868+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T04:08:16.902+0000] {logging_mixin.py:154} INFO - [2023-12-18T04:08:16.902+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T04:08:16.915+0000] {logging_mixin.py:154} INFO - [2023-12-18T04:08:16.914+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T04:08:16.939+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.091 seconds
[2023-12-18T04:12:10.046+0000] {processor.py:157} INFO - Started process (PID=1340) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T04:12:10.048+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T04:12:10.049+0000] {logging_mixin.py:154} INFO - [2023-12-18T04:12:10.048+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T04:12:10.057+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T04:12:10.259+0000] {logging_mixin.py:154} INFO - [2023-12-18T04:12:10.258+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T04:12:10.268+0000] {logging_mixin.py:154} INFO - [2023-12-18T04:12:10.267+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T04:12:10.288+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.245 seconds
[2023-12-18T04:15:59.874+0000] {processor.py:157} INFO - Started process (PID=1572) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T04:15:59.876+0000] {processor.py:829} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-18T04:15:59.877+0000] {logging_mixin.py:154} INFO - [2023-12-18T04:15:59.876+0000] {dagbag.py:536} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T04:15:59.889+0000] {processor.py:839} INFO - DAG(s) dict_keys(['example_local_kubernetes_executor']) retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-18T04:15:59.938+0000] {logging_mixin.py:154} INFO - [2023-12-18T04:15:59.938+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-12-18T04:15:59.953+0000] {logging_mixin.py:154} INFO - [2023-12-18T04:15:59.953+0000] {dag.py:3722} INFO - Setting next_dagrun for example_local_kubernetes_executor to None, run_after=None
[2023-12-18T04:15:59.979+0000] {processor.py:179} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.109 seconds
